cmake/Scripts/GenerateSleefHeader.cmake

```diff
@@ -19,23 +10,31 @@ if(OPTION_SHOW_CONFIG)
 endif(OPTION_SHOW_CONFIG)
 
 # Generate header for x86
+  set(run1 2 4 __m128d __m128 __m128i __m128i __SSE2__)
```

Comment:
I actually tried to come up with something, but it's just mkrename run parameters really and they're not used anywhere else. Also there is no need to differentiate between the variables or anything like that.